Changelog.md

```diff
@@ -4,6 +4,7 @@
 
 * Allow actors and time gates to deal with decimal percentages (https://github.com/jnunemaker/flipper/pull/492)
 * Change Flipper::Cloud::Middleware to receive webhooks at / in addition to /webhooks.
+* Add `write_through` option to ActiveSupportCacheStore adapter to support write-through caching (https://github.com/jnunemaker/flipper/pull/512)
```

Comment:
Not sure on the etiquette here, I can remove this if you prefer these to happen outside PRs.

Reply:
That's fine! You are an exemplary contributor. :)